packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components/NextButton/NextButton.spec.js

```diff
@@ -41,12 +41,12 @@ describe('NextButton', function () {
     })
 
     it('should not be disabled if props.disabled is false', function () {
-      expect(wrapper.find('WithTheme(Button)').props().disabled).to.be.false()
+      expect(wrapper.props().disabled).to.be.false()
     })
 
     it('should be disabled if props.disabled is true', function () {
       wrapper.setProps({ disabled: true })
-      expect(wrapper.find('WithTheme(Button)').props().disabled).to.be.true()
+      expect(wrapper.props().disabled).to.be.true()
```

Comment:
I'm not really sure what this is testing, since `props.disabled` is set to `true` on the previous line.